### DISCUSSION TITLE
Better indicate "Troubleshooting Information" heading in support pane is expandable

### DISCUSSION THREAD

**Discussion by partingscientist** [uBlockOrigin/uBlock-issues#2619] [Ideas] [answered]
<details>

<summary>In case it wasn't obvious, I am talking about this</summary>

![image](https://user-images.githubusercontent.com/115052854/235139339-0b01955f-fd2f-411f-9ec3-2804634539c7.png)

</details>

I recently learnt while skimming Reddit support posts that this heading is expandable, which would have saved time whenever I report filter issues due to my setup. I never asked support on Reddit before, which is why I have never learnt this to be a thing in the first place. In addition, this is not documented in the [wiki](https://github.com/gorhill/uBlock/wiki/The-%22Report-a-filter-issue%22-form). Finally, my cursor doesn't change into a pointer when hovering over this heading.

Would it be possible to better indicate that this heading is expandable? A naive approach would be simply adding "(click to expand)" or something similar, but I'm sure you can come up with a way better approach. Thank you.

**Comment by partingscientist** [comment, marked as answer]
Solved with https://github.com/gorhill/uBlock/commit/6d81614bd85037b87aced32a566da6d6a28d27e4. Thank you once again.

**Comment by gothic-bum** [comment]
@gorhill

Perhaps a bug with https://github.com/gorhill/uBlock/commit/6d81614bd85037b87aced32a566da6d6a28d27e4 , or with a recent change made to the troubleshooting information.

Using uBO 1.49.3b2 on Firefox 112.0.2, if any of the default filter lists are un-checked (e.g. uBlock filters, EasyList, EasyPrivacy, Peter Lowe, Online Malicious URL Blocklist), the troubleshooting information is completely blank even when clicked. It is also blank from the Support Pane in the Dashboard.

The issue does not occur with uBO 1.49.3b1.

**Reply by ghost** [reply]
I can confirm this behavior. I was setting up a new Firefox profile with uBO and making changes to both to customize it for my personal use and noticed that the Troubleshooting Information on the Support pane in the dashboard was completely blank. See the screenshot below. I didn't get the chance/time to narrow down the exact cause to one single change though.

Also, on a new, fresh profile and fresh installation of uBO everything is fine.

I am on Arch Linux using Firefox 112.0.2 & uBO 1.49.3b2.

`https://user-images.githubusercontent.com/124740436/235250097-f4b77254-9bcc-42d0-be2d-310c7eee758a.png`

**Reply by gorhill** [reply]
https://github.com/gorhill/uBlock/commit/536bfb33877afdc8f74f076d3ae55827a65a76aa

**Reply by ghost** [reply]
I can confirm this is fixed in uBO 1.49.3b3.

Thank you @gorhill! :+1:
